collie/cross_validation.py

```diff
@@ -239,13 +240,13 @@ def _stratified_split(interactions: BaseInteractions,
                       processes: int,
                       seed: int,
                       force_split: bool) -> Tuple[Interactions, Interactions]:
-    try:
-        if interactions.negative_sample_type == 'item':
-            users_or_items = interactions.mat.row
-        elif interactions.negative_sample_type == 'user':
-            users_or_items = interactions.mat.col
-    except AttributeError:
-        users_or_items = interactions.mat.row
+    users_or_items = interactions.mat.row
+    opposite_type = 'user'
+
+    if getattr(interactions, 'negative_sample_type', 'item') == 'user':
+        users_or_items = interactions.mat.col
+        opposite_type = 'item'
```

Comment:
Should we do the same `row/col` naming scheme here too? My preference is to eliminate as many `user_or_item` variable names as possible.